client/index.production.html

```diff
@@ -70,6 +70,8 @@
             // @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-3.0
             // @license-end
         </script>
-        <div id="app"></div>
+        <div id="app">
+            <div style="opacity: 0">{{.Html}}</div>
```

Comment:
What's this do?

Reply:
Helps search engine to index pages faster and without Js. It's invisible to end-users and removed once Js loads